app/src/main/java/com/mashup/mobalmobal/ui/profile/domain/model/ProfileItemMapper.kt

```diff
@@ -0,0 +1,72 @@
+package com.mashup.mobalmobal.ui.profile.domain.model
+
+import com.mashup.mobalmobal.R
+import com.mashup.mobalmobal.ui.profile.data.dto.MyDonateDto
+import com.mashup.mobalmobal.ui.profile.data.dto.MyPostDto
+
+fun MyDonateDto.toProfileItems(headerStringId: Int): List<ProfileItem> {
+    return mutableListOf<ProfileItem>().also {
+        if (donates.isNullOrEmpty()) return emptyList()
+
+        it.add(
+            ProfileItem.Header(
+                titleId = headerStringId
+            )
+        )
+
+        it.addAll(
+            donates.map { donate ->
+                ProfileItem.Donation(
+                    donationId = donate.postId.toString(),
+                    imageUrl = donate.post.postImage ?: "",
+                    title = donate.post.title,
+                    description = donate.post.description ?: "",
+                    goalPrice = donate.post.goalPrice.toDouble(),
+                    donatedPrice = donate.post.currentAmount.toDouble(),
+                    startAt = donate.post.startedAt ?: "",
+                    endAt = donate.post.endAt ?: ""
+                )
+            }
+        )
+    }
+}
+
+fun MyPostDto.toProfileItems(headerStringId: Int): List<ProfileItem> {
+    return mutableListOf<ProfileItem>().also {
+        if (posts.isNullOrEmpty()) return emptyList()
+
+        it.add(
+            ProfileItem.Header(
+                titleId = headerStringId
+            )
+        )
+
+        it.addAll(
+            posts.map { post ->
+                ProfileItem.Donation(
+                    donationId = post.postId.toString(),
+                    imageUrl = post.postImage ?: "",
+                    title = post.title,
+                    description = post.description ?: "",
+                    goalPrice = post.goalPrice.toDouble(),
+                    donatedPrice = post.goalPrice.toDouble(),
+                    startAt = post.startedAt ?: "",
+                    endAt = post.endAt ?: ""
+                )
+            }
+        )
+    }
+}
+
+fun Triple<Int, Int, Int>.toSummaryProfileItem() =
```

Comment:
```suggestion
fun Triple<Int, Int, Int>.toSummaryProfileItems(): List<ProfileItem> =
```
반환형이 명시되어 있지 않으면 human error를 초래할 수 있을 것 같아요.
함수 명에서도 List형이 반환됨을 명시해 줘야 할 것 같아요

Reply:
함수명으로 유추할 수 있어서 생략했는데 human error 생각해서 넣어야겠네요